lib/appliance_console/external_httpd_configuration.rb

```diff
@@ -18,8 +19,8 @@ module ExternalHttpdConfiguration
       HTTP_KEYTAB         = "/etc/http.keytab"
 
       EXTERNAL_AUTH_FILE  = "conf.d/cfme-external-auth"
-      HTTPD_EXTERNAL_AUTH = "/etc/httpd/#{EXTERNAL_AUTH_FILE}"
-      HTTPD_CONFIG        = "/etc/httpd/conf.d/cfme-https-application.conf"
+      HTTPD_EXTERNAL_AUTH = Pathname.new(MiqApache.root_dir).join("etc/httpd/#{EXTERNAL_AUTH_FILE}")
+      HTTPD_CONFIG        = Pathname.new(MiqApache.root_dir).join("etc/httpd/conf.d/cfme-https-application.conf")
```

Comment:
@brandondunne @Fryguy @kbrock @jrafanie 
Today we might only be using the config directory but its possible in the future we might have to get to other directories like the modules directory. The root directory is either "/" or "/opt/rh/httpd24/root", from which we can derive the other directories, so we can either add separate directories for config/module or we can derive it from the root directory. I am ok with either approach. We can add extra methods in the miq_apache to add support for config directory which it can derive from the root directory but I am not sure if we should add extra environmental variables for storing directories for config/modules